Mage.Client/src/main/java/org/mage/card/arcane/ModernSplitCardRenderer.java

```diff
@@ -0,0 +1,329 @@
+package org.mage.card.arcane;
+
+import mage.ObjectColor;
+import mage.abilities.costs.mana.ManaCosts;
+import mage.constants.CardType;
+import mage.view.CardView;
+
+import java.awt.*;
+import java.awt.geom.Rectangle2D;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by StravantUser on 2017-03-30.
+ */
+public class ModernSplitCardRenderer extends ModernCardRenderer {
+
+    private class HalfCardProps {
+        int x, y, w, h, cw, ch;
+
+        String name;
+        String manaCostString;
+        ObjectColor color;
+        ArrayList<TextboxRule> rules = new ArrayList<>();
+        ArrayList<TextboxRule> keywords = new ArrayList<>();
+    }
+
+    private static ArrayList<CardType> ONLY_LAND_TYPE = new ArrayList<CardType>() {{add(CardType.LAND);}};
+
+    // Right and left halves of the card content
+    private HalfCardProps rightHalf = new HalfCardProps();
+    private HalfCardProps leftHalf = new HalfCardProps();
+
+    // Where and how big is the divider between the card halves
+    private int dividerAt;
+    private int dividerSize;
+
+    // Is fuse / aftermath
+    private boolean isFuse = false;
+    private boolean isAftermath = false;
+
+    public ModernSplitCardRenderer(CardView view, boolean isTransformed) {
+        super(view, isTransformed);
+
+        rightHalf.manaCostString = ManaSymbols.getStringManaCost(cardView.getRightSplitCosts().getSymbols());
+        leftHalf.manaCostString = ManaSymbols.getStringManaCost(cardView.getLeftSplitCosts().getSymbols());
+
+        rightHalf.color = getColorFromManaCostHack(cardView.getRightSplitCosts());
+        leftHalf.color = getColorFromManaCostHack(cardView.getLeftSplitCosts());
+
+        parseRules(view.getRightSplitRules(), rightHalf.keywords, rightHalf.rules);
+        parseRules(view.getLeftSplitRules(), leftHalf.keywords, leftHalf.rules);
+
+        rightHalf.name = cardView.getRightSplitName();
+        leftHalf.name = cardView.getLeftSplitName();
+
+        for (String rule: view.getRules()) {
+            if (rule.contains("Fuse")) {
+                isFuse = true;
+                break;
+            }
+        }
```

Comment:
not sure if view.getRules() returns an array on a list, but in case it returns a list, then I like the stream().anyMatch() more